### ISSUE TITLE
Bug: TypeError: errorCallSites.splice is not a function

### ISSUE THREAD

**Issue by deman4ik** [fullstack-build/tslog#29]
**Describe the bug**
Got `TypeError: errorCallSites.splice is not a function` while trying to log `uncaughtException`.

**To Reproduce**

```ts
import { Logger } from "tslog";

const logger = new Logger();

function logError(err: any) {
    console.error(err);  // GOOD -> TypeError: Cannot read property 'id' of undefined
    logger.error(err); // !ERROR HERE -> TypeError: errorCallSites.splice is not a function
}

process.on("uncaughtException", logError);

let obj: any;
const id = obj.id; // generating uncaught exception
```

**Expected behavior**
Log error `TypeError: Cannot read property 'id' of undefined`

**Additional context**
Node.JS 12.18.0
typescript ^3.9.6
tslog ^2.6.0

**Comment by terehov** [comment]
Thank you, I was able to reproduce it. I guess it's also related to #27 
Should be able to come up with a fix shortly.

**Comment by terehov** [comment]
I have just released a patch version 2.6.1 that should fix your problem. Let me know if it works. 👍 


<img width="1792" alt="image" src="https://user-images.githubusercontent.com/490885/86854617-085cbe80-c0b9-11ea-8224-5a1bcfe0564f.png">


https://github.com/fullstack-build/tslog/releases/tag/v2.6.1
